### DISCUSSION TITLE
Cannot Initialize Kernel with any environment

### DISCUSSION THREAD

**Discussion by danistenia** [microsoft/vscode-jupyter#10985] [Questions and Answers] [answered]
Hi. 
This problem is driving me crazy. I do not know what has it change since last time, maybe it is a MacOS update on my Monterrey Macbook Pro model. Thing is, this is the error I got on every single jupyter notebook I try to start:

![image](https://user-images.githubusercontent.com/55037866/181995257-580eceed-4884-47d2-b143-3dd49f2e187b.png)
 
I have tried several approaches without any luck, if someone could help will be awesome. I think is something about the path which is taking homebrew/Caskroom something like that.
Here I got a list of my envs:

![image](https://user-images.githubusercontent.com/55037866/181995320-9dbe2386-7032-4f48-b33a-c4921f8509ea.png)

On the other hand if I tried to initialize jupyter notebook from any env I can run it without any problem. So I think it something wrong with Visual. 

Regards,
Daniel

**Comment by DonJayamanne** [comment, marked as answer]
Thanks for filing this issue, looks like `libssl` hasn't been installed properly on your machine. Or is the wrong bitness.
I am using a Mac with the silicon chip and not running into this, you might have some issues with the installation of Python on your machine.

Please try the following:
* Open a terminal outside vscode and confirm you can run the following without any issues:
```shell
> conda activate base_bice_anaconda # first activate the conda environment 
> conda install -c conda-forge notebook # Install jupyter notebooks into the conda environment
> python -m jupyter notebook
```
* If that works, please ensure you select the `base_bice_anaconda` environment in the following command
`Jupyter: Select Interpreter to Start Jupyter Server`
* Reload VS Code and try running a cell once again, & please send the logs again if that fails, use the command `Jupyter; Show Output` to get the logs

**Reply by danistenia** [reply]
Oh my god that was it!!! Thank you very much! 
I was so sad because I was religate it to use Jupyter Notebook and it was like coming back to the past.

<img width="1138" alt="image" src="https://user-images.githubusercontent.com/55037866/182057399-54f9a9d9-972f-4577-89c9-ac4e24aaba5f.png">
